EIPS/eip-4974.md

```diff
@@ -1,7 +1,7 @@
 ---
 eip: 4974
-title: Experience (EXP) Token Standard
-description: A standard interface for fungible, non-tradable tokens, also known as EXP.
+title: Rating
+description: A standard interface for ratings.
```

Comment:
```suggestion
title: Comparable Non-Transferable Tokens
description: ...
```
Not sure what a good description for this would be that doesn't duplicate the title. It can't include the word `standard` or similar words.